CTP/README.md

```diff
@@ -122,40 +124,65 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
   
 - **SHELL**
   - Prepare

```

Comment:
As is,
When you want to write your own test case
To be,
When you plan to write your own test case